test/integration/test/custom_vocabularies.test.js

```diff
@@ -0,0 +1,32 @@
+const clientHelper = require('../src/client-helper');
+const configHelper = require('../src/config-helper');
+const CustomVocabularyStatus = require('../../../dist/src/models/CustomVocabularyStatus').CustomVocabularyStatus;
+const client = clientHelper.getCustomVocabulariesClient(configHelper.getApiKey());
+
+test('Can submit custom vocabulary', async () => {
+    const informationSubmit = await client.submitCustomVocabularies([{phrases:['some','custom','vocabularies']}]);
+
+    expect(informationSubmit.status).toBe(CustomVocabularyStatus.InProgress);
+    expect(informationSubmit.created_on).not.toBeNull();
+    expect(informationSubmit.id).not.toBeNull();
+    expect(informationSubmit.failure).toBeUndefined();
+});
+
+test('Can retreive submitted custom vocabulary', async () => {
```

Comment:
Add done argument so the test will wait until it's been called
```suggestion
test('Can retreive submitted custom vocabulary', async (done) => {
```